src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -57,6 +59,13 @@ export function isJupyterNotebook(option: NotebookDocument | string) {
     }
 }
 
+export function isJupyterKernel(kernel?: VSCNotebookKernel): kernel is VSCodeNotebookKernelMetadata {
```

Comment:
I think this is cleaner than writing this code everywhere.
@IanMatthewHuff (made me realize in our conversation that this could be improved by moving into a simple function).

Reply:
This is an unrelated change in the PR